reward/doc.go

```diff
@@ -14,5 +14,62 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
 
-// Package reward implements the Klaytn reward system.
+/*
+Reward Package
+Package Reward implements the Klaytn Reward System.
+Klaytn Reward System manages stakingInfo and distributes block reward.
+
+Managing stakingInfo
+Klaytn uses weightedRandom policy to choose a block proposer.
+It means, the percentage of becoming a block proposer depends on how much klay a node has staked.
```

Comment:
Why don't you describe more detail where staked klay is?

Reply:
klay --> KLAY

Reply:
reflected